Content.Server/ADT/Chat/ChatSystem.ADT.cs

```diff
@@ -28,26 +28,28 @@ private Dictionary<ICommonSession, ICChatRecipientData> GetWhisperRecipients(Ent
                 continue;
 
             var observer = ghostHearing.HasComponent(playerEntity);
+            var entClearRange = clearRange;
+            var entMuffledRange = muffledRange;
 
```

Comment:
```suggestion

            var entClearRange = clearRange;
            var entMuffledRange = muffledRange;
            
            var observer = ghostHearing.HasComponent(playerEntity);
            if (observer)
            {
                recipients.Add(player, new ICChatRecipientData(-1, true));
                continue;
            }

```

снизу  if (observer) и перенести сюда
небольщая оптимизейшен, сокращающая вычисления